lib/modules/dosomething/dosomething_api/resources/campaign_resource.inc

```diff
@@ -271,14 +271,14 @@ function _campaign_resource_retrieve($nid) {
 /**
  * Retrieve and show index list response of campaigns requested.
  *
- * @param string $ids
- * @param bool $staff_pick
- * @param bool $mobile_app
- * @param string $mobile_app_date
- * @param string $term_ids
- * @param int $count
- * @param bool $random
- * @param int $page
+ * @param  string  $ids
```

Comment:
One other concern I'd have is this messes with diffs (i.e. you may have to change all lines for indentation when you only _really_ changed one parameter). I don't have particularly strong feelings either way, but something to consider! :)